notebooks/link_prediction/Overlap-Similarity.ipynb

```diff
@@ -421,18 +330,9 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 11,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "CPU times: user 25.4 ms, sys: 896 µs, total: 26.3 ms\n",
-      "Wall time: 24.5 ms\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "%%time\n",
```

Comment:
There's a `WARNING: name 'time' is not defined` showing up in this notebook and jaccard

Reply:
I can take time out, I could be something in the environment

Reply:
fixed